service/worker/service.go

```diff
@@ -487,7 +504,12 @@ func (svc *Service) handleNodeBindingUpdates(ctx context.Context, newBindingPair
 				}
 			}(ctx)
 
-			svc.pushHandlers[newBinding.Key()] = handler
+			// store only if subscriber creation succeeds
+			error := <-notifyCh
```

Comment:
if start errors out, can we design stop in a way that we handle cases, we shouldn't need notify channel seperately